### ISSUE TITLE
[Bug]: Python error comes up after some startups

### ISSUE THREAD

**Issue by lionblaze55** [bottlesdevs/Bottles#2139]
### Describe the bug

Not sure how this is being triggered, but I am getting the following crash report occasionally when starting bottles. Will be monitoring to see if I can nail down what behavior exactly is triggering it. Happening on both my 36 and 37 Silverblue installs.

```
'arguments'  File "/app/share/bottles/bottles/frontend/utils/threading.py", line 63, in __target
    result = self.task_func(*args, **kwargs)

  File "/app/share/bottles/bottles/frontend/widgets/program.py", line 180, in _run
    WineExecutor.run_program(self.config, self.program, with_terminal)

  File "/app/share/bottles/bottles/backend/wine/executor.py", line 121, in run_program
    args=program["arguments"],
```

### To Reproduce

start application.

### Package

Flatpak from Flathub

### Distribution

Fedora 37 Silverblue

### Troubleshooting Logs

```shell
Version: 2022.9.28.1
Display:
    X.org: true
    X.org (port): :0
    Wayland: true
Graphics:
    vendors:
        intel:
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.19.13-300.fc37.x86_64
Distro:
    Name: GNOME
    Version: '43'
Disk:
    Total: 8251387904
    Free: 8251236352
RAM:
    MemTotal: "16.5\_GB"
    MemAvailable: "11.7\_GB"
Bottles_envs: null
```

### Display Protocol

Wayland

### Display Backend used by Bottles

Wayland

**Comment by davidmartos96** [comment]
I believe this was fixed by 604ccb9909f3ce469383656dd01b68dbc757a918

**Comment by mirkobrombin** [comment]
It should be fixed by that commit. I was not able to replicate so I think it's safe to close the issue and re-open if the problem persist with the next release.
